auth0/v3/management/custom_domains.py

```diff
@@ -0,0 +1,76 @@
+from .rest import RestClient
+
+
+class CustomDomains(object):
+
+    """Auth0 custom domains endpoints
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self, id=None):
+        url = 'https://%s/api/v2/custom-domains' % self.domain
+        if id is not None:
+            return url + '/' + id
+        return url
+
+    def get_all(self):
```

Comment:
This is typically called `all`:

https://github.com/auth0/auth0-python/blob/master/auth0/v3/management/clients.py#L70
https://github.com/auth0/auth0-python/blob/master/auth0/v3/management/rules.py#L27